src/coreclr/nativeaot/System.Private.CoreLib/src/System/Runtime/RuntimeImports.cs

```diff
@@ -845,6 +850,307 @@ ushort ClrInstanceID
         [RuntimeImport(RuntimeLibrary, "RhpCheckedXchg")]
         internal static extern object InterlockedExchange([NotNullIfNotNull(nameof(value))] ref object? location1, object? value);
 
+#if TARGET_X86
+        [Intrinsic]
+        [LibraryImport(RuntimeImports.RuntimeLibrary)]
+        [UnmanagedCallConv(CallConvs = [typeof(CallConvCdecl)])]
+        [SuppressGCTransition]
```

Comment:
```suggestion
        [UnmanagedCallConv(CallConvs = [typeof(CallConvCdecl), typeof(SuppressGCTransition)])]
```
Nit: This can be one-liner. (Either way is fine.)